extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -195,4 +177,70 @@ export class DeployConfigPage extends DacFxConfigPage {
 		this.databaseLoader.loading = false;
 		return true;
 	}
+
+	/*
+	Function that is used to change the radio button DOM based on whether databases exist or not.
+	*/
+	private async disableUpgradeRadioButton(): Promise<azdata.FormComponent> {
+		/* Set the upgrade radio button to be disabled and call the updateNewRadioButton function
+		to update the new radio button accordingly.
+		*/
+		this.upgradeRadioButton.enabled = false;
+		this.newRadioButton.checked = true;
+
+		this.updateNewRadioButton();
+
+		return this.displayRadioButtons(this.upgradeRadioButton, this.newRadioButton);
+	}
+
+	/*
+	Function that is used to update the window if upgrade radio button is selected.
+	*/
+	private async updateUpgradeRadioButton(): Promise<void> {
+		this.model.upgradeExisting = true;
+		this.formBuilder.removeFormItem(this.databaseComponent);
+		this.formBuilder.addFormItem(this.databaseDropdownComponent, { horizontal: true, componentWidth: 400 });
+		this.model.database = (<azdata.CategoryValue>this.databaseDropdown.value).name;
+
+		// add deploy plan page and remove and re-add summary page so that it has the correct page number
+		this.instance.wizard.removePage(DeployNewOperationPath.summary);
+		let deployPlanPage = this.instance.pages.get(PageName.deployPlan);
+		let summaryPage = this.instance.pages.get(PageName.summary);
+		this.instance.wizard.addPage(deployPlanPage.wizardPage, DeployOperationPath.deployPlan);
+		this.instance.wizard.addPage(summaryPage.wizardPage, DeployOperationPath.summary);
+	}
+
+	/*
+	Function that is used to update the window if new radio button is selected.
+	*/
+	private async updateNewRadioButton(): Promise<void> {
+		this.model.upgradeExisting = false;
+		this.formBuilder.removeFormItem(this.databaseDropdownComponent);
+		this.formBuilder.addFormItem(this.databaseComponent, { horizontal: true, componentWidth: 400 });
+		this.model.database = this.databaseTextBox.value;
+		this.instance.setDoneButton(Operation.deploy);
+
+		// remove deploy plan page and read summary page so that it has the correct page number
+		this.instance.wizard.removePage(DeployOperationPath.summary);
+		this.instance.wizard.removePage(DeployOperationPath.deployPlan);
+		let summaryPage = this.instance.pages.get(PageName.summary);
+		this.instance.wizard.addPage(summaryPage.wizardPage, DeployNewOperationPath.summary);
+	}
+
+	/*
+	Function to display the radio buttons on the window.
+	*/
+	private async displayRadioButtons(upgradeRadioButton: azdata.RadioButtonComponent, newRadioButton: azdata.RadioButtonComponent): Promise<azdata.FormComponent> {
```

Comment:
change name to createRadioButtonFlexContainer. They actually don't get displayed until they're added to the formBuilder and that gets initialized

Reply:
Acknowledged, thank you!